src/Accounts/Accounts/Account/ConnectAzureRmAccount.cs

```diff
@@ -608,9 +620,9 @@ private string PreProcessAuthScope()
             return mappedScope;
         }
 
-        private bool IsUsingInteractiveAuthentication()
+        private bool IsBrowserPopUpInteractiveAuthentication()
         {
-            return ParameterSetName == UserParameterSet && UseDeviceAuthentication == false;
+            return ParameterSetName.Equals(UserParameterSet) && UseDeviceAuthentication.IsPresent == false;
```

Comment:
What is the difference from "ParameterSetName == UserParameterSet && UseDeviceAuthentication == false"